lib/output.js

```diff
@@ -92,6 +92,8 @@ function toFile (fileOut, callback) {
 function toBuffer (options, callback) {
   if (is.object(options)) {
     this._setBooleanOption('resolveWithObject', options.resolveWithObject);
+  } else if (this.options.resolveWithObject) {
+    this._setBooleanOption('resolveWithObject', false);
```

Comment:
Can we get away with just setting `this.options.resolveWithObject = false` in the else clause?

Reply:
I kept the save structure than the previous statement but it's more to check the input than a setter. Also change to a simple assignment.